api/python/quilt3/packages.py

```diff
@@ -1229,6 +1231,15 @@ def physical_key_is_temp_file(pk):
             self._set(lk, pkg[lk])
 
         pkg._build(name, registry=registry, message=message)
+
+        navigator_url = get_from_config("navigator_url")
+        registry_parsed = PhysicalKey.from_url(registry)
+        shorthash = Package._shorten_tophash(name, registry_parsed, pkg.top_hash)
+        print(f"Package {name}@{shorthash} pushed to s3://{dest_parsed.bucket}")
+        if navigator_url is None:
```

Comment:
I think we need slightly different logic. I believe that there is both a config template and by default that we try to configure to `open.quiltdata.com`. Adding @kevinemoore for review and advice. Was able to test the navigator case and it works fine.